worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -1112,10 +1139,6 @@ namespace RTC
 			  origSsrc,
 			  origSeq,
 			  origTimestamp);
-
-#ifdef MS_RTC_LOGGER_RTP
-			packet->logger.Dropped(RtcLogger::RtpPacket::DropReason::SEND_RTP_STREAM_DISCARDED);
```

Comment:
Same, this packet is discarded, hence `packet->logger.Dropped()` must be called

Reply:
That log was only present in SimulcastConsumer and not in the very same code block in other Consumers. Why?